nsv13/code/modules/overmap/physics.dm

```diff
@@ -512,6 +512,34 @@ The while loop runs at a programatic level and is thus separated from any thrott
 		return ..()
 	// if a bump is that fast then it's not a bump. It's a collision.
 	if(istype(A, /obj/structure/overmap) && c_response)
+		// If the target is an asteroid
+		if ( istype( A, /obj/structure/overmap/asteroid ) )
+			// If the attacker hits the rock hard enough
+			if ( bump_velocity >= 3 )
+				// If the attacker is a player ship, and not a mining ship
+				if ( istype( B, /obj/structure/overmap/nanotrasen ) && !istype( B, /obj/structure/overmap/nanotrasen/mining_cruiser ) )
```

Comment:
AI ships are also /obj/structure/overmap/nanotrasen, so this would also affect them, maybe a better way would be checking if B.role == MAIN_OVERMAP

Reply:
Which then could also get rid of the AI check on 523, though this is probably fine as it is.